geode-core/src/test/java/org/apache/geode/pdx/PdxSerializableDUnitTest.java

```diff
@@ -107,6 +94,33 @@ public Object call() throws Exception {
     });
   }
 
+  private static String TEST_REGION_NAME = "testSimplePdx";;
```

Comment:
should move this to the top of the class and make it final.

Reply:
Done.